dist/css/styles.css

```diff
@@ -10194,7 +10222,7 @@ header.masthead .masthead-heading {
   position: absolute;
   width: 100%;
   height: 100%;
-  background: rgba(254, 209, 54, 0.9);
+  background: rgb(130, 216, 207, 0.7);
```

Comment:
From rgb(a) - the a is used for opacity, if you remove the 'a', the last parameter, 0.7 will be ignored